yare-model-converters/src/test/java/com/sabre/oss/yare/common/converter/aliases/TypeAliasesTest.java

```diff
@@ -67,12 +70,82 @@ void shouldInitializeMapOfAliasesByNameProperly() {
                         getEntryWithNameKey("ZonedDateTime", ZonedDateTime.class),
                         getEntryWithNameKey("List", List.class),
                         getEntryWithNameKey("Map", Map.class),
-                        getEntryWithNameKey("Set", Set.class));
+                        getEntryWithNameKey("Set", HashSet.class),
+                        getEntryWithNameKey("CustomType", CustomType.class));
     }
 
     @Test
     void shouldInitializeMapOfAliasesByTypeProperly() {
-        //given /when /then
+        //given
+        TypeAliases typeAliases = new TypeAliases();
+        // /when /then
+        assertThat(typeAliases.getAliasesMappedByType().entrySet())
+                .containsExactlyInAnyOrder(
+                        getEntryWithTypeKey("Object", Object.class),
+                        getEntryWithTypeKey("String", String.class),
+                        getEntryWithTypeKey("Integer", Integer.class),
+                        getEntryWithTypeKey("Long", Long.class),
+                        getEntryWithTypeKey("Double", Double.class),
+                        getEntryWithTypeKey("Boolean", Boolean.class),
+                        getEntryWithTypeKey("Byte", Byte.class),
+                        getEntryWithTypeKey("Short", Short.class),
+                        getEntryWithTypeKey("Character", Character.class),
+                        getEntryWithTypeKey("Float", Float.class),
+                        getEntryWithTypeKey("Void", Void.class),
+                        getEntryWithTypeKey("int", int.class),
+                        getEntryWithTypeKey("long", long.class),
+                        getEntryWithTypeKey("double", double.class),
+                        getEntryWithTypeKey("boolean", boolean.class),
+                        getEntryWithTypeKey("byte", byte.class),
+                        getEntryWithTypeKey("short", short.class),
+                        getEntryWithTypeKey("char", char.class),
+                        getEntryWithTypeKey("float", float.class),
+                        getEntryWithTypeKey("void", void.class),
+                        getEntryWithTypeKey("ZonedDateTime", ZonedDateTime.class),
+                        getEntryWithTypeKey("List", List.class),
+                        getEntryWithTypeKey("Map", Map.class),
+                        getEntryWithTypeKey("Set", HashSet.class),
+                        getEntryWithTypeKey("CustomType", CustomType.class));
+    }
+
+    @Test
+    void shouldInitializeMapOfAliasesByNameProperlyWhenNoCustomOverridingFileExists() {
+        //given
+        TypeAliases typeAliases = new TypeAliases(ALIAS_CONFIGURATION_RESOURCE_FILE, "notExisting-typeAliases-custom.properties");
+        // /when /then
+        assertThat(typeAliases.getAliasesMappedByName().entrySet())
```

Comment:
I think it is not needed to check here all aliases introduced by default file (only check one or two).